src/constants.ts

```diff
@@ -18,18 +18,22 @@ export const MIME_TYPES = {
 };
 
 export const TEXT_EDITOR_TOOLBAR = [
-  [
-    { header: [1, 2, 3, 4, 5, 6, false] },
-    { font: [] },
-    'bold',
-    'italic',
-    'underline',
-    'strike',
-    { color: [] }, // default colors depending on theme
-    { background: [] }, // default colors depending on theme
-  ],
-  [{ align: [] }],
-  [{ list: 'ordered' }, { list: 'bullet' }, 'code-block', 'link', 'formula'],
+    [
+		[{ header: [1, 2, 3, 4, 5, 6, false] }],
+		[{ font: [] }],
+		['bold',
+		'italic',
+		'underline',
+		'strike'],
+		[{ color: [] }], // default colors depending on theme
+		[{ background: [] }], // default colors depending on theme
+		[{ align: [] }],
+		[{ list: 'ordered' }, { list: 'bullet' }, 'code-block', 'link', 'formula'],
+		['emoji'],
+	],
+	"emoji-toolbar": true,
+    "emoji-textarea": true,
+    "emoji-shortname": true,
```

Comment:
When I run yarn storybook i have this error 
![image](https://user-images.githubusercontent.com/79811264/200306731-611ffb20-287c-424f-b88c-2c4c852864b4.png)

Reply:
I think you have some troubles because of Windows.
You should look into WSL, a UNIX Ccommand line for Windows, and run the project using it.

Reply:
Hello, I try it on linux machine, but same problem. But I made the code. Therefore, I am asking you for a compilation on your side. Because I can't see where the error could be. Thanks a lot

Reply:
The error you currently have is dependency related. Delete your yarn.lock and node_modules, then run `yarn` again from WSL.

From my side, the lines I've highlighted are false because of the syntax. You **cannot** write 
```js
[
{ "some": "content" },
"emoji-toolbar": true,
    "emoji-textarea": true,
    "emoji-shortname": true,
]
```
It should look like the following. **But even with the following changes, it might still not work the way react-quill should.** 
```js
[
{ "some": "content" },
{ "emoji-toolbar": true,
    "emoji-textarea": true,
    "emoji-shortname": true,}
]
```